template-skeleton/template/src/components/common/Map.js

```diff
@@ -6,6 +6,11 @@ import { makeStyles, useTheme, useMediaQuery } from '@material-ui/core';
 import { setViewState } from '@carto/react-redux';
 import { BASEMAPS, GoogleMap } from '@carto/react-basemaps';
 
+const MAPS_TYPES = {
```

Comment:
Maybe BASEMAP_TYPES, instead of MAPS_TYPES

Reply:
Done